sdk/go/api_esc_extensions.go

```diff
@@ -0,0 +1,287 @@
+package esc
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"gopkg.in/ghodss/yaml.v1"
+)
+
+type EscClient struct {
+	rawClient *RawAPIClient
+	EscAPI    *EscAPIService
+}
+
+func NewClient(cfg *Configuration) *EscClient {
+	client := &EscClient{rawClient: NewRawAPIClient(cfg)}
+	client.EscAPI = client.rawClient.EscAPI
+	return client
+}
+
+func (c *EscClient) ListEnvironments(ctx context.Context, org string, continuationToken *string) (*OrgEnvironments, error) {
+	request := c.EscAPI.ListEnvironments(ctx, org)
+	if continuationToken != nil {
+		request = request.ContinuationToken(*continuationToken)
+	}
+
+	envs, _, err := request.Execute()
+	return envs, err
+}
+
+func (c *EscClient) GetEnvironment(ctx context.Context, org, envName string) (*EnvironmentDefinition, string, error) {
+	env, resp, err := c.EscAPI.GetEnvironment(ctx, org, envName).Execute()
+	if err != nil {
+		return nil, "", err
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, "", err
+	}
```

Comment:
What is the contents of `body` here vs. the contents of `env`? Is `body` the raw text?

Also, I notice that `EnvironmentDefinition` only contains `json` struct tags--it seems unlikely to unmarshal properly given that most definitions will not be valid JSON. Is that a code generator issue?

Reply:
Yes, body in case people just want to work with the text (run regex or grep or something).

So far I have not seen an issue with the marshalling, but it is possible I am not taxing it enough.  Any example you can think of to test that?  This is why there is a raw Yaml version as well.